c7n/resources/ecs.py

```diff
@@ -513,6 +506,31 @@ def augment(self, resources):
         return results
 
 
+class ConfigECSTaskDefinition(ConfigSource):
+
+    def load_resource(self, item):
+        resource = super().load_resource(item)
```

Comment:
the keys on all of the attribute are off. we probably just want to just unconditionally remap them for top level elements and containers array.

Reply:
more specifically config returns ContainerAttributes and we want containerAttributes to match describe output.

Reply:
thanks for the feedback @kapilt. Yeah, that was something I was thinking about but wasn't sure. I'll make the changes shortly

Reply:
Remaped top level keys

Reply:
it should be a recursive helper function else it wont get to container definitions which aren't top level keys as their in the containers array.

Reply:
Sounds good!

Reply:
i went ahead and added this